components/MegaMenu.vue

```diff
@@ -6,16 +6,39 @@
       :title="category.content.name"
       :link="localePath(getCatLink(category))"
     >
-      <SfList>
-        <SfListItem v-for="child in category.childrenCategories" :key="child.id">
-          <SfMenuItem :label="$t(child.content.name)" :link="localePath(getCatLink(child))" />
-        </SfListItem>
-      </SfList>
+      <div class="sf-mega-menu-option">
+        <div class="flex-grow-3">
+          <SfList class="outer-list">
+            <SfListItem v-for="child in category.childrenCategories" :key="child.id">
+              <SfMenuItem class="sf-title" :label="child.content.name" />
```

Comment:
use child?.content?.name insted of child.content.name

Reply:
Getting compilation error while using the child?.content?.name.

Reply:
@TriptiVermaXP add a category getter, getName = (category) => category?.content?.name

Also, Add a category getter for categoryCode getCategoryCode = (category) => category?.categoryCode
And use that for line 12, :key=getCategoryCode because child.id === null

Reply:
Done